src/components/Form.js

```diff
@@ -0,0 +1,36 @@
+import Button from './Button'
+import Input from './Input'
+import PropTypes from 'prop-types'
+
+
+function Form({userInput, setUserInput, setCounty}) {
+    
+    function handleChange(event) {
+        setUserInput(event.target.value);
+    }
+
+    function handleSubmitCounty(event) {
+        event.preventDefault()
+        setCounty(userInput)
+        setUserInput('')
+        event.target.focus()
+    }
+    
+return (
+            <form onSubmit={handleSubmitCounty}>
+                <Input 
+                    value={userInput}
+                    onChange= {handleChange} 
+                    placeholderText="Gib deinen Landkreis ein..."/>
+                <Button text="Suchen"/>
```

Comment:
Why do you use the ellipsis in your placeholder text. This text in comibnation with your button text, is basically your CTA. Thus, I would formulate it more determinedly. "Gib hier Deinen Landkreis ein." (as placeholder text) "Jetzt den Inzidenzwert erfahren." (as the button text) - or something like that :)